packages/react-button/src/components/Button/Button.types.ts

```diff
@@ -1,4 +1,4 @@
-import { ARIAButtonShorthandProps } from '@fluentui/react-aria';
+import type { ARIAButtonShorthandProps } from '@fluentui/react-aria';
```

Comment:
It looks like you also have this `import type` refactoring in #20891. Probably best to revert it in this PR (ditto in a bunch of files).